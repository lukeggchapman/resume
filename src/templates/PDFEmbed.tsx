import React from 'react'
import { css } from '@emotion/core'

export interface PDFEmbedTemplateProps {
  pageContext: {
    pdfPublicURL: string
  }
}

const PDFEmbedTemplate = ({ pageContext }: PDFEmbedTemplateProps) => (
  <embed
    css={css`
      width: 100%;
      height: 100%;
    `}
    src={pageContext.pdfPublicURL}
  />
)

export default PDFEmbedTemplate
