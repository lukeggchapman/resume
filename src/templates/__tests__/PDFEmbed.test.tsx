import React from 'react'
import { shallow } from 'enzyme'

import PDFEmbed, { PDFEmbedTemplateProps } from '../PDFEmbed'

const baseProps: PDFEmbedTemplateProps = {
  pageContext: {
    pdfPublicURL: 'pdfPublicURLTest',
  },
}

describe('PDFEmbed', () => {
  const render = () => shallow(<PDFEmbed {...baseProps} />)

  it('renders correctly', () => {
    expect(render()).toMatchSnapshot()
  })
})
