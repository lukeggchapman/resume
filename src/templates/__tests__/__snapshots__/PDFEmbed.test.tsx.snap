// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PDFEmbed renders correctly 1`] = `
<embed
  css={
    Object {
      "map": undefined,
      "name": "bggj8i",
      "next": undefined,
      "styles": "
      width: 100%;
      height: 100%;
    ",
    }
  }
  src="pdfPublicURLTest"
/>
`;
